src/test/resources/io/inversion/cloud/action/sql/upsert002/01-POST-customers.json

```diff
@@ -0,0 +1,10 @@
+{
+	"lastName": "Tester1",
```

Comment:
Indention is incorrect.

Reply:
what is wrong with it? my formatter thinks it is correct...and looks right here.

Reply:
I see it as having an indentation of 6 spaces.  The other files I see as having 3 or 4.  Just stood out to me.